packages/loot-core/src/server/accounts/sync.js

```diff
@@ -366,7 +370,8 @@ async function normalizeNordigenTransactions(transactions, acctId) {
         payee: trans.payee,
         account: trans.account,
         date: trans.date,
-        notes: trans.remittanceInformationUnstructured,
+        notes: trans.remittanceInformationUnstructured
+          || (trans.remittanceInformationUnstructured || []).join(', '),
```

Comment:
Shouldn't this be `remittanceInformationUnstructuredArray`?

Reply:
Yep, copy-paste error... Will fix it along with any lint issues in a moment.